vendor/brave-ios/BATBraveRewards.h

```diff
@@ -18,6 +18,8 @@ NS_SWIFT_NAME(BraveRewardsConfiguration)
 //@property (nonatomic, getter=isDebug) BOOL debug;
 /// The rewards environment
 @property (nonatomic) BATEnvironment environment;
+/// The rewards build channel
+@property (nonatomic) BATBraveAdsBuildChannel *buildChannel;
```

Comment:
Since this is not provided in the initializer, `nullable` annotation should be added to property declaration

Reply:
Fixed